static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.js

```diff
@@ -24,7 +25,12 @@ const fillAdvertSlots = (): Promise<void> => {
     return Promise.all(dependencies).then(() => {
         // Get all ad slots
         const adverts = qwery(dfpEnv.adSlotSelector)
-            .filter(adSlot => !(adSlot.id in dfpEnv.advertIds))
+            // TODO: more general solution for omitting the epic ad slot here
+            .filter(
+                adSlot =>
+                    !(adSlot.id in dfpEnv.advertIds) &&
+                    adSlot.id !== epicAdSlotId
+            )
```

Comment:
I would rather avoid changing the logic of DFP 🙈 Sorry!

Can we add the Epic to the page dynamically, as we do with comment adverts, then fill it? That would stop us from changing the behaviour of DFP

Reply:
I think now that I'm loading the ad slot correctly, this is no longer needed (race condition avoided).